docs/jsonapi-swr/overview.md

```diff
@@ -217,6 +209,66 @@ const { data: user } = useDatx(getUserQuery(id));
 const { data: todo } = useDatx(getTodoByUserQuery(user));
 ```
 
+### Query naming convention
+
+Query can be a static object or a function. Both cases should be suffixed with `query` to make it clear that it's a query.
+If you use a function, it's recommended to add `get` prefix to indicate that it's a getter.
+Query could also be lazy initializer function that returns a query. In this case it's recommended to use `get` prefix as well.
+
+> Rule of thumb is to use `get` prefix if query depends on some other data. 
+
+> Lazy initializer function is SWR concept used for conditional fetching. You can find more details in [SWR Conditional Fetching](https://swr.vercel.app/docs/conditional-fetching) documentation.
+
+```ts
+// collection query
+const todosQuery = {
+  op: 'getMany',
+  type: 'todos',
+} as const;
+
+// single item query
+const getTodoQuery = (id?: string) =>
+  id
+    ? ({
+        id,
+        op: 'getOne',
+        type: 'todos',
+      } as const satisfies IGetOneExpression<typeof Todo>)
+    : null;
+
+// single item lazy query, a.k.a currying
+const getTodoQuery = (id?: string) => () =>
+  id
+    ? ({
+        id,
+        op: 'getOne',
+        type: 'todos',
+      } as const satisfies IGetOneExpression<typeof Todo>)
+    : null;
```

Comment:
As discussed, I would remove this in favor of using the `useSWR` API. `useDatx(() => getTodoQuery(todo.id))`